### ISSUE TITLE
Lost translations in 0.84

### ISSUE THREAD

**Issue by Anonym-tsk** [home-assistant/home-assistant-polymer#2292]
**Home Assistant release with the issue:** 0.84.2

**Last working Home Assistant release (if known):** 0.83.3

**Browser and Operating System:** MacOS, Android, latest Google Chrome

**Description of problem:**
After updating, a lot of strings lost
![](http://monosnap.css3.su/Home_Assistant_2018-12-13_14-48-47.png)
![](http://monosnap.css3.su/Home_Assistant_2018-12-13_14-49-06.png)
![](http://monosnap.css3.su/Home_Assistant_2018-12-13_14-49-50.png)

**Comment by sgttrs** [comment]
I have the same issue. And it happens random

**Comment by balloob** [comment]
When you change tab and back, are they fixed?

**Comment by Anonym-tsk** [comment]
@balloob No

**Comment by Anonym-tsk** [comment]
In 0.84.3 problem still not solved.

**Comment by Skaronator** [comment]
Yup can confirm, 0.84.3 doesn't fix this issue. I've also cleared browser cache to be sure, that I'm on the latest frontend version.

**Comment by iantrich** [comment]
Open a new issue and reference this one rather than commenting on a closed issue, please
